src/core/sagas/user.js

```diff
@@ -0,0 +1,29 @@
+// Disabled because of
+// https://github.com/benmosher/eslint-plugin-import/issues/793
+/* eslint-disable import/order */
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+/* eslint-enable import/order */
+
+import { SET_AUTH_TOKEN } from 'core/constants';
+import { getState } from 'core/sagas/utils';
+import { userProfileLoaded } from 'core/reducers/user';
+import { userProfile as userProfileApi } from 'core/api/user';
+
+export function* fetchUserProfile({ payload }) {
+  const { token } = payload;
+
+  const state = yield select(getState);
+
+  const response = yield call(userProfileApi, {
```

Comment:
I think this should have a `try/catch` around it... at least that's what we do for all of our other sagas and then we use `yield put(errorHandler.createErrorAction(error));` to dispatch an error.

See something like the search saga: https://github.com/mozilla/addons-frontend/blob/master/src/core/sagas/search.js#L21

Reply:
Yeah I know how this works, but the UI gives the error handler ID, which is not possible here. That's why @kumar303 suggested to bubble up the errors: https://github.com/mozilla/addons-frontend/pull/2989#discussion_r134889238.

Reply:
Ahhhh, right. Okay, can you add a comment in here explaining why this saga is different than usual then and why we aren't using `errorHandler`? Sorry I missed some of the comments here.

Reply:
Yes, just updated the code.